src/model/herramientas/Pico.java

```diff
@@ -0,0 +1,36 @@
+package model.herramientas;
+
+import model.materiales.Madera;
+import model.materiales.Material;
+import model.materiales.Metal;
+import model.materiales.Piedra;
+
+public class Pico extends Herramienta {
+
+    public Pico(Material material){
+        super(material);
```

Comment:
este super seria como hacer directo this.material = material ? que esa linea es la del constructor del que hereda